nemo/collections/asr/modules/rnnt.py

```diff
@@ -50,6 +50,525 @@
 from nemo.utils import logging
 
 
+class StatelessNet(torch.nn.Module):
+    def __init__(self, context_size, vocab_size, emb_dim, blank_idx, blank_as_pad, normalization_mode, dropout):
+        super().__init__()
+        self.context_size = context_size
```

Comment:
Add docstring for arguments. Preferably with defaults suggestions. Add inline types to the init function signature.

Reply:
Also assert context size is positive int or 0.

Reply:
Done.